engine/src/star_system_jump.cpp

```diff
@@ -47,10 +47,15 @@
 #include "root_generic/options.h"
 
 void CacheJumpStar(bool destroy) {
-    static Animation *cachedani = new Animation(game_options()->jumpgate.c_str(), true, .1, MIPMAP, false);
+    static Animation* cached_ani;
```

Comment:
I'm pretty sure this is slower, so why is it better?
Looking through the code, we only call `CacheJumpStar` once, in `Universe::Init`.
It does not look as if we are ever getting to the cleanup section.

Looking further, `cachedani` isn't accessible. I wonder if `Animation` is doing something weird inside or we can just assume they wrote something sane for once and just delete the whole thing. 

If deleting this breaks something, we can just delete the function and move 

`static Animation *cachedani = new Animation(game_options()->jumpgate.c_str(), true, .1, MIPMAP, false);`

to Universe::Init.

Reply:
Again, I made these changes to allow `cachedani` / `cached_ani` to be static, and initialized only once, after the game config is loaded. (I also renamed the variable to be easier to read.)

I think `Animation` must be doing something weird inside. At any rate, I'm not prepared to delete this whole section in this PR. Maybe later.